osu.Game/Beatmaps/Drawables/DifficultyIconTooltip.cs

```diff
@@ -70,8 +153,55 @@ public void SetContent(DifficultyIconTooltipContent content)
 
             displayedContent = content;
 
+            // Header row
             starRating.Current.BindTarget = displayedContent.Difficulty;
             difficultyName.Text = displayedContent.BeatmapInfo.DifficultyName;
+
+            // Don't show difficulty stats if showExtendedTooltip is false
+            if (!displayedContent.ShowExtendedTooltip)
+            {
+                difficultyFillFlowContainer.Hide();
+                miscFillFlowContainer.Hide();
+                return;
+            }
+
+            // Show the difficulty stats if showExtendedTooltip is true
+            difficultyFillFlowContainer.Show();
+            miscFillFlowContainer.Show();
+
+            double rate = 1;
+
+            if (displayedContent.Mods != null)
+            {
+                foreach (var mod in displayedContent.Mods.OfType<IApplicableToRate>())
+                    rate = mod.ApplyToRate(0, rate);
+            }
+
+            double bpmAdjusted = displayedContent.BeatmapInfo.BPM * rate;
+
+            BeatmapDifficulty originalDifficulty = new BeatmapDifficulty(displayedContent.BeatmapInfo.Difficulty);
+
+            if (displayedContent.Mods != null)
+            {
+                foreach (var mod in displayedContent.Mods.OfType<IApplicableToDifficulty>())
+                {
+                    mod.ApplyToDifficulty(originalDifficulty);
+                }
+            }
+
+            Ruleset ruleset = displayedContent.Ruleset.CreateInstance();
+            BeatmapDifficulty adjustedDifficulty = ruleset.GetRateAdjustedDisplayDifficulty(originalDifficulty, rate);
+
+            // Difficulty row
+            circleSize.Text = "CS: " + adjustedDifficulty.CircleSize.ToString("0.##");
+            drainRate.Text = " HP: " + adjustedDifficulty.DrainRate.ToString("0.##");
+            approachRate.Text = " AR: " + adjustedDifficulty.ApproachRate.ToString("0.##");
+            overallDifficulty.Text = " OD: " + adjustedDifficulty.OverallDifficulty.ToString("0.##");
+
+            // Misc row
+            length.Text = "Length: " + TimeSpan.FromMilliseconds(displayedContent.BeatmapInfo.Length / rate).ToString("mm\\:ss");
+            bpm.Text = " BPM: " + Math.Round(bpmAdjusted, 0);
+            maxCombo.Text = " Max Combo: " + displayedContent.BeatmapInfo.TotalObjectCount;
```

Comment:
This is not correct.

Reply:
Honestly looks fine without it. Guess I'll just remove it.
<img width="287" alt="Screenshot 2024-01-22 at 8 50 11 AM" src="https://github.com/ppy/osu/assets/69545310/a48a0503-d00a-4f1d-8b9f-d84ceedcb399">

Reply:
> I don't really feel like implementing it here

it's like two inches away

https://github.com/ppy/osu/blob/49c117793c7424fbbac10057f33f4b7d60a57fab/osu.Game/Beatmaps/StarDifficulty.cs#L19-L22

Reply:
> > I don't really feel like implementing it here
> 
> it's like two inches away
> 
> https://github.com/ppy/osu/blob/49c117793c7424fbbac10057f33f4b7d60a57fab/osu.Game/Beatmaps/StarDifficulty.cs#L19-L22

I remember trying to use that, but it didn't work so I used objectcount thinking it was the same.
<img width="295" alt="image" src="https://github.com/ppy/osu/assets/69545310/5c1c5a17-db65-43f5-8b9c-5b0576cbeec6">
`maxCombo.Text = " Max Combo: " + displayedContent.Difficulty.Value.MaxCombo;`